src/secrets.ts.txt

```diff
@@ -1,3 +0,0 @@
-// Create a secrets.ts file with this format:
-export const clientId: string = "";
-export const clientSecret: string = "";
```

Comment:
I'm in favor of eventually deleting this placeholder file, but before we do that we need to either
- Update the documentation so it's obvious how to create and format `secrets.ts`
- Use a different secret management solution like extension preferences (my preferred)

Reply:
Usually, in executable codes with config files and stuff, it is a good practice to create a `filename.extension.example` (`secrets.ts.example` in this case) file that contains the info about how to add your keys, etc. And the best part about this being that there are VSCode extensions made specifically for these `example` files to help easier setup.

I personally would be against the first suggestion as instead of creating a new file in a specified format, an end-user (or a developer) will find it more convenient to just update the two string values in a placeholder file.

I think we can create the `.example` thing to go with conventions for the time being till the second suggestion by you can be implemented